Assignment5Objects/src/assignment5objects/MainProgram.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * @author Samantha Shurie
+ * Assignment 5: Working with Objects
+ */
+package assignment5objects;
+
+
+public class MainProgram {
+   
+    /**
+     * Main method implements both the RetailItem and Car classes. This method
+     * creates a new instance of the Car class and increases/decreases it's speed
+     * field. It also creates three instances of the RetailItem class and prints
+     * their fields.
+     * @param args 
+     */
+    public static void main(String[] args) {
+        int currentSpeed = 0;   //used to retrieve speed value of car object
+         
+        println("EXERCISE 2: CAR CLASS\n");
+        
+        Car car = new Car(2010, "Honda");
+
+        println("\tAccelerating...");
+
+        for(int i = 0; i < 5; i++) {
+            car.accelerate();
+            currentSpeed = car.getSpeed();
+            println("\t\tSpeed is now " + currentSpeed + ".");
+        }
+
+        println("\n\tApplying Brakes...");
+
+        for(int i = 0; i < 5; i++) {
+            car.brake();
+            currentSpeed = car.getSpeed();
+            println("\t\tSpeed is now " + currentSpeed + ".");
+        }
```

Comment:
Alternatively instead of using the same variable for each of these loops you could try something like this
```java
int decelerationSpeed = 0;
for(int i = 0; i < 5; i++) {
  car.brake();
  decelerationSpeed = car.getSpeed();
  println("\t\tSpeed is now " + decelerationSpeed + ".");
}
```